Dockerfile

```diff
@@ -1,29 +1,22 @@
-ARG VERSION=latest
-ARG OS=debian
-ARG CLI_NAME=atmos
-
-FROM cloudposse/geodesic:$VERSION-$OS
+# syntax=docker/dockerfile:1
+FROM alpine:3.19 as deps
 
 # Install terraform
-ARG TF_1_VERSION=1.2.3
-RUN apt-get update && \
-    apt-get install -y -u --allow-downgrades terraform-1="${TF_1_VERSION}-*"
-
-# Set Terraform 1.x as the default `terraform` in this container.
-# Other versions of TF can be installed + used via Spacelift before_init scripts.
-RUN update-alternatives --set terraform /usr/share/terraform/1/bin/terraform
+ARG TF_1_VERSION=1.5.7 # last release under Mozilla Public License
+ARG ALPINE_VERSION=1.52.0
```

Comment:
I'm assuming you meant this to be `ATMOS_VERSION` and then to use it below? Let's update before merge.

Reply:
you're right, nice catch. fixed